src/stepDefinitions/webdavContext.js

```diff
@@ -133,47 +132,6 @@ Then(/^as "([^"]*)" (file|folder) "([^"]*)" should exist inside folder "([^"]*)"
   return fileOrFolderShouldExist(user, path.join(folder, file), resourceType)
 })
 
-Then('as {string} the last uploaded folder should exist', function(userId) {
-  return fileOrFolderShouldExist(userId, client.sessionId, 'folder')
-})
-
-Then(
-  'as {string} the last uploaded folder should contain the following files inside the sub-folders:',
-  async function(user, files) {
-    files = files.raw().map(item => item[0])
-
-    const sessionId = client.sessionId
-    const response = await webdavHelper.propfind(
-      `/files/${user}/${sessionId}`,
-      user,
-      [],
-      'infinity'
-    )
-    const result = xml2js(response, { compact: true })
-
-    const elements = _.get(result, 'd:multistatus.d:response')
-    if (elements === undefined) {
-      throw new Error('Received unexpected response:\n' + result)
-    }
-
-    const uploadedFilesHref = elements.map(elem => _.get(elem, 'd:href._text')).filter(item => item)
-
-    const regexToExtractBaseName = RegExp(`/${sessionId}/upload[0-9]+file/(.*)`)
-    const uploadedFilesWithBaseName = uploadedFilesHref
-      // unmatched items return null, else is array
-      .map(file => _.nth(file.match(regexToExtractBaseName), 1))
-      .filter(item => item)
-
-    const filesNotFoundInUploads = _.difference(files, uploadedFilesWithBaseName)
-    assert.strictEqual(
-      filesNotFoundInUploads.length,
-      0,
-      'Could not find following files inside sub-folders of the session folder\n' +
-        filesNotFoundInUploads
-    )
-  }
-)
-
```

Comment:
These steps are removed because it requires `sessionId` from web and it's browser-specific. These steps are kept in the web's `webdavContext.js` so no need to keep it here as well